### DISCUSSION TITLE
圣遗物评分是不是寄了

### DISCUSSION THREAD

**Discussion by DongGua-Jun** [Arondight/Adachi-BOT#632] [General]
如题

**Comment by mark9804** [comment]
测试过可以正常使用，请提供更多信息

**Reply by Arondight** [reply]
总之就是很难用……门栓拍屏，很难保证高识别率

**Comment by DongGua-Jun** [comment]
![QQ截图20220204191729](https://user-images.githubusercontent.com/75875483/152520186-6298c550-f02f-478f-b7f3-02c12ff35168.png)

腾讯云的windows环境，更详细的log好像看不到
